runtime/runtime_darwin.go

```diff
@@ -26,6 +26,8 @@ func (user *OSUser) CreateNew(okIfExists bool) (err error) {
 		fullname="${0}"
 		password="${1}"
 		echo "Creating user '${username}' with home directory '${homedir}' and password '${password}'..."
+		maxid=$(dscl . -list '/Users' 'UniqueID' | awk '{print $2}' | sort -ug | tail -1)
```

Comment:
does `sort` need `-n`?

Reply:
looks like `sort -n` and `sort -g` are [roughly equivalent](https://askubuntu.com/questions/840781/sort-command-g-versus-n-flag) but `-n` is more common, so maybe more clear

Reply:
That should be covered by the `-g`:
```
     -g, --general-numeric-sort, --sort=general-numeric
             Sort by general numerical value.  As opposed to -n, this option handles general floating points.  It has a more permissive format than that allowed by -n but it has a significant per-
             formance drawback.
```

Perhaps `-n` would have been enough, but `-g` also covers it.

Reply:
Looks like pretty similar performance:

```
pmoore@Petes-iMac:~/go/src/github.com/taskcluster/generic-worker/worker_types master $ time for ((x=0; x<100;x++)); do dscl . -list '/Users' 'UniqueID' | awk '{print $2}' | sort -ug >/dev/null; done

real	0m3.442s
user	0m2.385s
sys	0m1.676s
pmoore@Petes-iMac:~/go/src/github.com/taskcluster/generic-worker/worker_types master $ time for ((x=0; x<100;x++)); do dscl . -list '/Users' 'UniqueID' | awk '{print $2}' | sort -un >/dev/null; done

real	0m3.405s
user	0m2.349s
sys	0m1.610s
```